package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Collection of devtools packages",
   "scripts": {
     "flow": "flow",
-    "lint": "yarn run lint-css -s; yarn run lint-js -s",
+    "lint": "npm run lint-css -s; a=$?; npm run lint-js -s; b=$?; test $a -eq 0 -a $b -eq 0",
```

Comment:
This was the initial goal of the patch: use npm instead of yarn because it's always there (at least we can reasonnably assume it's globally installed).

If we use yarn, we need to either install it as devDependency or require it installed globally. I thought that to simply run scripts, npm is good enough, so no need to add an additional dependency or force the user to install it.

It's especially an issue for the "lint" script because it's run when pushing to a branch. I don't really mind for "nom" :)